src/api/db/migrate/20180731124926_create_status_checks.rb

```diff
@@ -0,0 +1,14 @@
+class CreateStatusChecks < ActiveRecord::Migration[5.2]
+  def change
+    create_table :status_checks do |t|
```

Comment:
You need to add `id: :integer` otherwise this will be a bigint and the verify migration tasks fails